src/runtime/entitlements-manager.js

```diff
@@ -465,6 +470,23 @@ export class EntitlementsManager {
     return this.responsePromise_.then(() => Promise.resolve(this.article_));
   }
 
+  /**
+   * The experiment flags that are returned by the article endpoint should be accessible from here.
+   * @returns {Promise<Array<string>>}
+   */
+  getExperimentConfigFlags() {
+    return this.getArticle().then((article) => {
+      const expConfig = article['experimentConfig'];
+      if (expConfig != null) {
+        const expFlags = expConfig['experimentFlags'];
+        if (expFlags != null) {
+          return expFlags;
+        }
+        return [];
```

Comment:
Should the method return `[]` if `expConfig` is `null` ?

Reply:
Ah yes, good catch! Updated & added another test

Reply:
Thanks!